packages/dashboard/src/app/views/exploreTemplates/content/templateGridView.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+  useFocusOut,
+  useMemo,
+} from '@web-stories-wp/react';
+import { __ } from '@web-stories-wp/i18n';
+import { trackEvent } from '@web-stories-wp/tracking';
+import { useGridViewKeys } from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_WRAPPER } from '../../../../constants';
+import { CardGrid } from '../../../../components';
+import {
+  PageSizePropType,
+  TemplatesPropType,
+  TemplateActionsPropType,
+} from '../../../../types';
+import { useConfig } from '../../../config';
+import { resolveRoute } from '../../../router';
+import TemplateGridItem, { FOCUS_TEMPLATE_CLASS } from './templateGridItem';
+
+const GridContainer = styled(CardGrid)`
+  width: calc(100% - ${PAGE_WRAPPER.GUTTER}px);
+`;
```

Comment:
I'm guessing this is legacy, but `CardGrid` doesn't actually take a `className` prop, so none of these styles get applied. Anyway, probably part of cleanup for another time, but just wanted to mention.

Reply:
👍  I can go ahead and update this here. It's definitely legacy at this point, no need to let it fester.

Reply:
Actually, I'm not seeing `className` for `CardGrid` anymore, can you be more specific?

Reply:
yeaup exactly. `CardGrid` doesn't take a `className` prop anymore so when you so `styled(CardGrid)(/*styles*/)` none of those styles actually get applied.

so right now:
```
// This is no different than just using CardGrid cus none of these styles get applied
const GridContainer = styled(CardGrid)`
  // This style doesn't actually get applied right now
  width: calc(100% - ${PAGE_WRAPPER.GUTTER}px);
`;
```

Reply:
oh yup! thanks!